packages/react-static-plugin-css-modules/README.md

```diff
@@ -0,0 +1,35 @@
+# react-static-plugin-css-modules
+
+A [React-Static](https://react-static.js.org) plugin that adds loader and SSR support for [css modules](https://github.com/css-modules/css-modules)
+
+## Installation
+
+In an existing react-static site run:
+
+```bash
+$ yarn add react-static-plugin-css-modules
+```
+
+Then add the plugin to your `static.config.js`:
+
+```javascript
+export default {
+  plugins: ["react-static-plugin-css-modules"]
+};
+```
+
+## With Options
+
+```javascript
+export default {
+  plugins: [
+    [
+      "react-static-plugin-css-modules",
+      {
+        includePaths: ["..."] // always includes `src/`
+        // other options for the sass-loader
```

Comment:
Yeah I need to change the wording and also update the pass through options. It’s 2am here tho, I don’t want to push and screw it up because I’m burnt out. 

I’ll be sure to follow through on your reviews, I know there’s more polish needed for sure